pom.xml

```diff
@@ -52,18 +54,34 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>${org.mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
Read about properties in pom files. 
You didn't declare the alias of liquibase.version in properties, so you are using not existing version. 

Also, remove the version here as spring-boot-starter-parent already includes liquibase

Reply:
not fixed